Jenkinsfile

```diff
@@ -21,8 +12,22 @@ pipeline {
                 }
             }
             stages {
+                stage('set_version_build') {
+                    when { not { branch "master" } }
+                    steps {
+                        sh './bumpversion.sh build'
+                    }
+                }
+                stage('set_version_release') {
+                    when { branch "master" }
+                    steps {
+                        sh './bumpversion.sh release'
+                    }
+                }
                 stage('package') {
                     steps {
+                        sh 'git submodule sync'
```

Comment:
This should be a separate step right at the start, before set version. There's no point bumping the version and then failing because of a git update!

Reply:
@lwgordonimos done